app-i18n/fcitx-gtk/fcitx-gtk-5.1.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake gnome2-utils xdg
+
+MY_PN="fcitx5-gtk"
+S="${WORKDIR}/${MY_PN}-${PV}"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}.tar.xz -> ${P}.tar.xz"
+KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
+
+DESCRIPTION="Gtk im module for fcitx5 and glib based dbus client library"
+HOMEPAGE="https://github.com/fcitx/fcitx5-gtk"
+
+LICENSE="LGPL-2.1+"
+SLOT="5"
+IUSE="gtk2 +gtk3 +gtk4 +introspection +snooper onlyplugin wayland"
+REQUIRED_USE="|| ( gtk2 gtk3 gtk4 )"
+
+RDEPEND="
+	app-i18n/fcitx:5
+	kde-frameworks/extra-cmake-modules:0
+	gtk2? ( x11-libs/gtk+:2 )
+	gtk3? ( x11-libs/gtk+:3[wayland?] )
+	gtk4? ( gui-libs/gtk:4[wayland?] )
+	introspection? ( dev-libs/gobject-introspection )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_LIBDIR="${EPREFIX}/usr/$(get_libdir)"
+		-DCMAKE_INSTALL_SYSCONFDIR="${EPREFIX}/etc"
+		-DCMAKE_BUILD_TYPE=Release
```

Comment:
Cmake eclass should set it to RelWithDebInfo. Is there a rationale for overriding it here (and elsewhere)?